libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1370,14 +1370,14 @@ export class RushConfiguration {
    *
    * @returns A map of dependency name --\> version specifier for implicitly preferred versions.
    */
-  public getImplicitlyPreferredVersions(): Map<string, string> {
+  public getImplicitlyPreferredVersions(subspace: Subspace): Map<string, string> {
```

Comment:
```suggestion
  public getImplicitlyPreferredVersions(subspace?: Subspace): Map<string, string> {
```

To make this not a breaking change.